integrator-analytics/integrator/conf/synapse.properties

```diff
@@ -60,14 +60,19 @@ synapse.debugger.port.command=9005
 synapse.debugger.port.event=9006
 
 # Configuration to enable mediation flow analytics
-mediation.flow.statistics.enable=true
-mediation.flow.statistics.tracer.collect.payloads=true
-mediation.flow.statistics.tracer.collect.properties=true
+mediation.flow.statistics.enable=false
```

Comment:
If these parameters should be kept as they are in the product pack, we do not need to add this file to templates.